packages/core/src/amazonqFeatureDev/session/sessionState.ts

```diff
@@ -367,7 +379,9 @@ export class CodeGenState extends CodeGenBase implements SessionState {
                     this.deletedFiles,
                     this.references,
                     this.tabID,
-                    this.currentIteration + 1
+                    this.currentIteration + 1,
```

Comment:
curious: there is any risk that currentIteration be different from those counts? And if its, have we tested what is shown to the user?

Reply:
Ideally the currentIteration would be simply "totaliteration-remainingIteration". This is calculated in the front end. 
However we decided to expose and use the API carried values since they are got from the source of truth - our ddb table.

Reply:
OK, so do we need remove currentIteration since we now have new counts?

Reply:
And this field `currentIteration` is not actually being used for now. I think the idea is to use iterationCounts instead.